synapse/daemon.py

```diff
@@ -200,7 +145,7 @@ def listen(self, url, **opts):
         # TODO: SSL
         ssl = None
 
-        server = s_glob.plex.listen(host, port, self._onLinkInit, ssl=ssl)
+        server = await s_glob.plex.listen(host, port, self._onLinkInit, ssl=ssl)
```

Comment:
do we listen on the global plex or on self.loop ?

Reply:
this will be addressed in the plex loop elimination